core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -6235,7 +6235,6 @@ private SqlNode navigationInDefine(SqlNode node, String alpha) {
         break;
       case 2:
         assert op.allowsNullTreatment();
-        assert op.requiresOver();
```

Comment:
does the comment above this if block have to be updated?

Reply:
Good catch, updated